hamocc/mo_apply_fedep.F90

```diff
@@ -62,9 +62,10 @@ subroutine apply_fedep(kpie,kpje,kpke,pddpo,omask,dust)
     do j = 1,kpje
       do i = 1,kpie
         if(omask(i,j) > 0.5) then
-          dustinp = dust(i,j) / 30. * dtb / pddpo(i,j,1)
-          ocetra(i,j,1,ifdust) = ocetra(i,j,1,ifdust) + dustinp
-          ocetra(i,j,1,iiron) = ocetra(i,j,1,iiron) + dustinp * perc_diron
+          dustflx(i,j,itdust)  = dust(i,j) / 30. * dtb / pddpo(i,j,1)
+          dustflx(i,j,isfe)    = dust(i,j) / 30. * dtb / pddpo(i,j,1) * perc_diron
```

Comment:
Could be simplified to `dustflx(i,j,isfe)  = dustflx(i,j,itdust)*perc_diron` (and in the next step, when exchanging the input, vice versa). Maybe add a comment on unit convesrion here (i.e. it is percent-iron  * kmol/kg factor, which is included in perc_diron).